Tests/FluentMySQLDriverTests/FluentMySQLDriverTests.swift

```diff
@@ -165,6 +165,97 @@ final class FluentMySQLDriverTests: XCTestCase {
         }
     }
 
+    func testBoolFilter() throws {
+        final class Clarity: Model {
+            static let schema = "clarities"
+
+            @ID(custom: .id, generatedBy: .database)
+            var id: Int?
+
+            @Field(key: "rain")
+            var rain: Bool
+
+            init() { }
+
+            init(rain: Bool) {
+                self.rain = rain
+            }
+        }
+
+        struct CreateClarity: Migration {
+            func prepare(on database: Database) -> EventLoopFuture<Void> {
+                return database.schema("clarities")
+                    .field("id", .int, .identifier(auto: true))
+                    .field("rain", .bool, .required)
```

Comment:
Not sure why this is failing...from the tests I can see Fluent is using the `BIT` type and from what I can tell `MySQLData` supports bools serializing to/from `BIT` correctly.

https://github.com/vapor/mysql-nio/blob/master/Sources/MySQLNIO/MySQLData.swift#L57
https://github.com/vapor/mysql-nio/blob/master/Sources/MySQLNIO/MySQLData.swift#L273

Maybe MySQL isn't liking the BIT type? It looks like we might have been using TINY in Fluent 3. If so we might wanna revert back to that.

Reply:
Fixed this by having MySQL serialize bools as `TINYINT(1)`.